include/class_loader/class_loader.h

```diff
@@ -236,12 +252,66 @@ class ClassLoader
     }
   }
 
+  /// Generates an instance of loadable classes (i.e. class_loader).
+  /**
+   * It is not necessary for the user to call loadLibrary() as it will be
+   * invoked automatically.
+   * if the library is not yet loaded (which typically happens when in
```

Comment:
Nit: I think this is the continuation of the previous sentence and not a new one